pom.xml

```diff
@@ -51,13 +51,13 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
-        <cassandra.version>3.11.10</cassandra.version>
+        <cassandra.version>4.0.1</cassandra.version>
         <artifact.flavor.version>c3.11</artifact.flavor.version>
 
         <pmd-config-url>pmd-rules.xml</pmd-config-url>
 
         <commons-cli.version>1.4</commons-cli.version>
-        <chronicle-queue.version>4.16.5</chronicle-queue.version>
+        <chronicle-queue.version>5.20.123</chronicle-queue.version>
```

Comment:
I'm thinking it should be possible to remove this dependency declaration and just use whatever version Cassandra 4.x provides.

Reply:
Yes, that makes sens, I will try that.